Index: connect.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/connect.php b/connect.php
new file mode 100644
--- /dev/null	(date 1747906560788)
+++ b/connect.php	(date 1747906560788)
@@ -0,0 +1,14 @@
+<?php
+$host = 'fdb1028.awardspace.net';
+$db   = '4637952_webapp2';
+$user = '4637952_webapp2';
+$pass = 'reisbureau12';
+
+try {
+    $conn = new PDO("mysql:host=$host;dbname=$db", $user, $pass);
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+} catch (PDOException $e) {
+    die("Connectie mislukt: " . $e->getMessage());
+}
+?>
+<?php phpinfo(); ?>
Index: test.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test.php b/test.php
new file mode 100644
--- /dev/null	(date 1747906403120)
+++ b/test.php	(date 1747906403120)
@@ -0,0 +1,14 @@
+<?php
+global $conn;
+include("connect.php");
+
+
+$sql = "SELECT * FROM test";
+$result = $conn->query($sql);
+
+while ($row = $result->fetch_assoc()) {
+    echo $row['text'] . "<br>";
+}
+
+$conn->close();
+?>
\ No newline at end of file
